docs/new_model.md

```diff
@@ -5,12 +5,103 @@ version](https://badge.fury.io/py/tensor2tensor.svg)](https://badge.fury.io/py/t
 [![GitHub
 Issues](https://img.shields.io/github/issues/tensorflow/tensor2tensor.svg)](https://github.com/tensorflow/tensor2tensor/issues)
 [![Contributions
-welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg)](CONTRIBUTING.md)
+welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg)](https://github.com/tensorflow/CONTRIBUTING.md)
```

Comment:
This link does not exist. Use `../CONTRIBUTING.md` (or `https://github.com/tensorflow/tensor2tensor/blob/master/CONTRIBUTING.md` if absolute path is needed for some reason)